config.yaml

```diff
@@ -92,3 +92,14 @@ options:
 
       For more information about KubeletConfiguration, see upstream docs:
       https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/
+  open-ports:
+    default: "80/tcp,443/tcp"
+    type: string
+    description: |
+      Changing this parameter will cause all ports already opened to be closed
+      and ports on new config value to be opened. That will lead to some downtime
+      Comma-separated list of ports to be opened on each worker node. Ports will
+      be opened using juju's open-port command. Syntax for ports are:
+      PORT[/protocol] or PORT_1-PORT_2[/protocol] (range) or icmp
+      By default, tcp ports 80 and 443 are opened for ingress controller.
```

Comment:
If I change this config to "80/tcp,443/tcp,5000/tcp" will port 80 or port 443 experience downtime?

Reply:
Yes. Check my comments below.